tests/base-test.cc

```diff
@@ -364,6 +364,6 @@ int main() try {
 
   return 0;
 } catch (const marisa::Exception &ex) {
-  std::cerr << ex.what() << std::endl;
+  std::cerr << ex.what() << '\n';
```

Comment:
In the past, when blindly following clang-tidy, I've found that lack of flush is annoying and performance is not important for most uses of endl.

https://clang.llvm.org/extra/clang-tidy/checks/performance/avoid-endl.html

Reply:
By the way stdout/err always flush by default on every write, even without std::endl

Reply:
If I understand correctly, `cerr` and `cin` flush `cout`, but `cout` doesn't get automatically flushed.

https://en.cppreference.com/w/cpp/io/cout